packages/shylu/shylu_node/tacho/src/impl/Tacho_Solver_Impl.hpp

```diff
@@ -13,7 +13,8 @@ namespace Tacho {
   template<typename VT, typename ST>
   Solver<VT,ST>
   ::Solver()
-    : _m(0), _nnz(0),
+    : _transpose(0), _mode(0), _order_connected_graph_separately(0),
+      _m(0), _nnz(0),
       _ap(), _h_ap(), _aj(), _h_aj(),
```

Comment:
@bathmatt - there is a shylu/tacho file change mixed into the panzer changes. Did you intend to commit this? If so maybe @kyungjoo-kim can review these two lines? They look fine to me.

Reply:
This is not matt's modification I believe. Those changes seem to be a side effect from merging my PR to develop. Merging this as it is okay to me.